app/views/external_users/advocates/claims/_miscellaneous_fees_form_step.html.haml

```diff
@@ -3,7 +3,7 @@
 = content_for :page_title, flush: true do
   = t('.page_title')
 
-= render partial: 'external_users/claims/fees_shared_header', locals: {page_header: t('page_header', scope: locale_scope), page_hint: t('page_hint', scope: locale_scope)}
+= render partial: 'external_users/claims/fees_shared_header', locals: {page_header: t('page_header', scope: locale_scope), page_notice: t(miscellaneous_fees_notice(claim), scope: locale_scope), page_hint: t('page_hint', scope: locale_scope)}
```

Comment:
1. 🤔 play off between conditional here and the header below `page_notice.present?`
2.  🤔  naming wise the `page_notice` key is potentially reusable (though may never be), but the logic of miscellaneous_fees_notice is pretty specific, so maybe...
```
page_notice: t(unused_material_notice(claim), scope: locale_scope),
```
3.  🤔 
```suggestion
#a helper
def fee_shared_headings(locale_scope)
   fee_shared_headings = { page_header: t('page_header', scope: locale_scope),  page_hint: t('page_hint', scope: locale_scope) }
  fee_shared_headings.merge!(page_notice: t('unused_materials_notice', scope: locale_scope)} if unused_materials_applicable?
  end
  
def unused_materials_applicable?
  [(claim.final? || claim.transfer?),
    claim.clar?,
    ['Trial', 'Cracked Trial'].include?(claim.case_type.name)
     ].all?
end

# view
= render partial: 'external_users/claims/fees_shared_header', locals: fee_shared_headings(locale_scope)
```

Reply:
@jsugarman I have updated this along the lines of suggestion 3 in this commit: a580b9826b5df437fdfe795e8f1ca448385ec060

After committing it occurred to me that the name of the `CriminalLegalAidRemuneration2020` module is not quite right. Suggestions?

Reply:
yes, just noticed. 
CriminalLegalAidRemuneration2020 --> CriminalLegalAidReview2020